### PR TITLE
Od 94 choosing license

### PR DESCRIPTION
**Pull request by sivang**
license is chosen based on access level, using custom javascript in license.html.

### CODE REVIEW

File: ckanext/ed/templates/scheming/form_snippets/license.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    },
+    is_required=h.scheming_field_required(field),
+    )
+%}
+
+    {%- snippet 'scheming/form_snippets/help_text.html', field=field -%}
+{% endcall %}
+
+<script language="JavaScript">
+    document.getElementById('field-access_level').onclick = function()
```
Comment (zelima) [inline ckanext/ed/templates/scheming/form_snippets/license.html]: Lets move this into fanstatic folder, where all javascript codes belong https://github.com/CivicActions/ckanext-ed/tree/master/ckanext/ed/fanstatic/js and import it  like it's done here https://github.com/CivicActions/ckanext-ed/blob/master/ckanext/ed/templates/package/snippets/resources_download.html#L2

File: ckanext/ed/templates/scheming/form_snippets/license.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    {
+      sel_license = document.getElementById('field-license_id');
+      sel_accesslevel = document.getElementById('field-access_level');
+      sel_license.selectedIndex = sel_accesslevel.selectedIndex + 1;
+      label =  document.getElementById('select2-chosen-3');
+      label.textContent = sel_license.options[sel_accesslevel.options.selectedIndex+1].text;
+    }
+</script>
```
Comment (zelima) [inline ckanext/ed/templates/scheming/form_snippets/license.html]: Generally, it's hard to follow the logic here. I'm not sure what exactly this function does. Would be great to have a comment.

I have a feeling that stuff is hardcoded here...

File: ckanext/ed/templates/scheming/form_snippets/license.html (hunk 1 above)
Comment (sivang) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: I missed it, what is it about?

File: ckanext/ed/templates/scheming/form_snippets/license.html (hunk 2 above)
Comment (sivang) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: Basically, I just select the later field based on the first one, by an arbitrary method (+1 in this case) once we know the matching, I'll change this to a 'case' like logic.

Comment (zelima) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: So it's not actually doing anything, but is for demo only. Let's say that in comments

File: ckanext/ed/templates/scheming/form_snippets/license.html (hunk 1 above)
Comment (zelima) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: https://docs.ckan.org/en/2.8/theming/fanstatic.html#adding-css-and-javascript-files-using-fanstatic

Comment (sivang) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: regarding the location of this code, it is with @JGulic that we've decided to put it here, so we'll wait for her to come back and see what she says regarding moving it to `fantastic` folder.

File: ckanext/ed/templates/scheming/form_snippets/license.html (hunk 2 above)
Comment (sivang) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: it is hard coded as when me and @JGulic checked, there didn't seem to be a better way to achieve this. Hardcoded in the sense that field names are hardcoded for this particular JS interaction.

File: ckanext/ed/templates/scheming/form_snippets/license.html (hunk 1 above)
Comment (sivang) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: So I researched about fanstatic, I couldn't find docs where they explain how it works within jinja2. Also, it is important for this code snippet to be loaded on the widget to hook onto the on_change event of the first selector, also, it'd be hard to pass parameters unless we turn this into a snippet by it's own but then this makes sense only if we're going to use it more than in once place.

File: ckanext/ed/templates/scheming/form_snippets/license.html (hunk 2 above)
Comment (sivang) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: Perhaps I should change it to reflect the 'case' select instead of inc on the var to get to the right choice in the second select. inc on vars is a bit faster then doing a case match, which was my original rational behind this.

Comment (zelima) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: @sivang please say this in a comment. I'm not asking cause I'm curious, but for a future coder that will have hard times to read the code. :)

File: ckanext/ed/templates/scheming/form_snippets/license.html (hunk 1 above)
Comment (zelima) [reply ckanext/ed/templates/scheming/form_snippets/license.html]: OK, let's keep it than
